lte/gateway/deploy/agw_install_docker.sh

```diff
@@ -61,6 +86,22 @@ git clone "${GIT_URL}" /opt/magma
 cd /opt/magma || exit
 git checkout "$MAGMA_VERSION"
 
+# changing intefaces name
+sed -i 's/GRUB_CMDLINE_LINUX=""/GRUB_CMDLINE_LINUX="net.ifnames=0 biosdevname=0"/g' /etc/default/grub
+sed -i 's/ens5/eth0/g' /etc/netplan/50-cloud-init.yaml
+# changing interface name
+grub-mkconfig -o /boot/grub/grub.cfg
```

Comment:
We might want to use the porcelain command `update-grub2` (or `update-grub`) instead for convenience.